### PR TITLE
Adding standard files

### PR DESCRIPTION
**Pull request by hyandell**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*Issue #, if available:*
```
Comment (therapon) [inline .github/PULL_REQUEST_TEMPLATE.md]: IonJs uses formated commit messages to automate the release process and versioning. 
We'll have to see how to incorporate this template. 

See https://github.com/amzn/ion-js/blob/master/CONTRIBUTE.md#pull-requests

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* A reproducible test case or series of steps
+* The version of our code being used
+* Any modifications you've made relevant to the bug
+* Anything unusual about your environment or deployment
+
+
+## Contributing via Pull Requests
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+
+1. You are working against the latest source on the *master* branch.
```
Comment (therapon) [inline CONTRIBUTING.md]: IonJs currently has a different process in place. 
https://github.com/amzn/ion-js/blob/master/CONTRIBUTE.md#code

**Comment by hyandell** [comment]
They're good minimal process. Happy to merge them into what you have already :)

**Comment by almann** [comment]
Closing as I believe this is incorporated--please re-open if this is incorrect.

**Pull request closed without merging**